examples/ima-video-responsive.amp.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
```

Comment:
sure. created https://github.com/ampproject/amphtml/issues/9142 and added it to "IMA Launch project": https://github.com/ampproject/amphtml/projects/24